app/helpers/react_helper.rb

```diff
@@ -4,24 +4,26 @@ module ReactHelper
   def search_results_layout(search, params, vertical, affiliate, search_options)
     data = {
       additionalResults: search.govbox_set,
+      agencyName: agency_name(affiliate.agency),
```

Comment:
We were _almost_ fully alphabetized here, so I moved some unrelated/existing things around to keep that alphabetization.